src/plugins/explore/public/components/patterns_table/patterns_container.tsx

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { selectRows, selectTotalHits } from '../../application/utils/state_management/selectors';
+import { PatternItem, PatternsTable } from './patterns_table';
+import { COUNT_FIELD, PATTERNS_FIELD } from './constants';
+
+export const PatternsContainer = () => {
+  const rows = useSelector(selectRows); // TODO: use rows from patterns-appended query
+  const totalHits = useSelector(selectTotalHits);
+
+  // Convert rows to pattern items or use default if rows is undefined
+  const items: PatternItem[] = rows?.map((row) => ({
+    pattern: row._source[PATTERNS_FIELD],
+    ratio: row._source[COUNT_FIELD] / 2096, // TODO: pull from total hits
```

Comment:
q: rows should automatically be the new response from patterns query? does this need to wait for that query to return?

Reply:
I believe I'm going to pull a different redux selector than row which will be defined specifically for the patterns query. It would need to wait for that query to finish executing.